sbi/samplers/vi/__init__.py

```diff
@@ -0,0 +1,16 @@
+from .vi_pyro_flows import get_flow_builder, get_default_flows
+from .vi_divergence_optimizers import get_VI_method, get_default_VI_method
+from .vi_sampling import (
+    get_sampling_method,
+    get_default_sampling_methods,
+    get_sampling_method_parameters_doc,
+)
+from .vi_quality_controll import get_quality_metric
```

Comment:
please use the full import `from sbi.samplers.vi.vi_quality_controll import get_quality_metric `